CarInspection/src/WebCarInspection/Helpers/DateTagHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+using System;
+
+namespace WebCarInspection.Helpers
+{
+    public class DateTagHelper : TagHelper
+    {
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "div";
```

Comment:
It should have more correct name, for example - display-current-date
Name of class should be almost the save, but with TagHelper suffix.